homework3/Employee.java

```diff
@@ -0,0 +1,98 @@
+import javax.print.DocFlavor.STRING;
+
+public class Employee{
+    private String name;
+    private int basic;
+    private int equity;
+    private int hra;
+    private int gratuity;
+    private int performance_bonus;
+
+    public Employee(String name,int basic,int equity,int hra,int gratuity,int performance_bonus){
+        this.name = name;
+        this.basic =basic;
+        this.equity =equity;
+        this.hra =hra;
+        this.gratuity =gratuity;
+        this.performance_bonus =performance_bonus;
+
+    }
+
+    public void print_details(){
+        System.out.println(name + " " + basic + " " +equity + " " +hra + " " +gratuity + " " +performance_bonus);
+    }
+
+    public static Employee[] sort(Employee[] employee_list)
+    {
+        int n = employee_list.length;
+        for (int i = 0; i < n-1; i++)
+            for (int j = 0; j < n-i-1; j++)
+                if (employee_list[j].basic > employee_list[j+1].basic)
+                {
+                    Employee temp = employee_list[j];
+                    employee_list[j] = employee_list[j+1];
+                    employee_list[j+1] = temp;
+                }
+        
+                return employee_list;
```

Comment:
NO.. you had to store a few employee objects in an ArrayList and then sort based on a custom comparator